packages/node/src/integrations/utils/http.ts

```diff
@@ -0,0 +1,147 @@
+import { getCurrentHub } from '@sentry/core';
+import { Span } from '@sentry/types';
+import * as http from 'http';
+import { URL } from 'url';
+
+/**
+ * Checks whether given url points to Sentry server
+ * @param url url to verify
+ */
+export function isSentryRequest(url: string): boolean {
+  const client = getCurrentHub().getClient();
```

Comment:
```js
const host = getCurrentHub().getClient()?.getDsn();
return !dsn ? false : url.includes(dsn.host);
```

`url` will always be there, as `extractUrl` always returns at least _some_ string.

Reply:
Not mine, but yeah, that's a lot cleaner. Nice.